docs/openapi/components/schemas/common/CTPAT.yml

```diff
@@ -64,23 +64,34 @@ properties:
         $linkedData:
           term: iataCarrierCode
           '@id': https://onerecord.iata.org/cargo/Company#airlineCode
```

Comment:
I know this isn't part of this PR but this probably wants to reference ` https://onerecord.iata.org/Carrier#airlineCode` as company#airlinecode isn't a thing

Reply:
No, wait, `https://onerecord.iata.org/Carrier#airlineCode` _is_ a thing. The IATA ontology explicitly calls that out: 
![image](https://user-images.githubusercontent.com/34443212/226853633-5c5ddb1e-44eb-408d-9a72-fab011b4cac8.png)

There's just an in consistency with how the ontology is published. I called that out more than half a year ago on the IATA github: 
https://github.com/IATA-Cargo/ONE-Record/issues/149

I'm open to arguments for going with the double-# URI instead. But I feel pretty confident about sticking with my earlier choice: 
1. The IATA spec explicitly says the IRI is `https://onerecord.iata.org/Carrier#airlineCode` 
2. Double-# URIs are not illegal, but discouraged (second # must be interpreted as a literal), further indicating that's the bug and not IATA's actual intention.
Pls open a separate issue if you think this warrants further discussion; I don't think it does.